shiny/ui/_html_dependencies.py

```diff
@@ -1,14 +1,28 @@
 from __future__ import annotations
 
+import json
+from pathlib import Path
+
 from htmltools import HTML, HTMLDependency
 
 from ..html_dependencies import jquery_deps
 
 
 def bootstrap_deps() -> list[HTMLDependency]:
+    bootstrap_version_file = (
+        Path(__file__).parent.parent.resolve()
+        / "www"
+        / "shared"
+        / "bootstrap"
+        / "version.json"
+    )
+    with open(bootstrap_version_file) as bvf:
+        version_txt = bvf.read()
+        bootstrap_version = json.loads(version_txt).get("bootstrap_version", "5")
```

Comment:
The version information should be written to a .py file. Take a look at htmltools's `_version.py`.

Reply:
Also save r-shiny version to signify dep version